### PR TITLE
One_DataModel_Ready_2

### PR DESCRIPTION
**Pull request by yasta5**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] Ready
- [ ] In Hold - (Reason for hold)

### CODE REVIEW

File: Packs/BrocadeSwitch/ModelingRules/BrocadeSwitch_1_3/BrocadeSwitch_1_3.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from_xdr_version: 3.5.0
```
Comment (evisochek) [inline Packs/BrocadeSwitch/ModelingRules/BrocadeSwitch_1_3/BrocadeSwitch_1_3.yml:1]: @yasta5 @guytamir10 Remove from_xdr_version

File: Packs/CiscoASA/ModelingRules/CiscoASA_1_3/CiscoASA_1_3.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+from_xdr_version: 3.5.0
```
Comment (evisochek) [inline Packs/CiscoASA/ModelingRules/CiscoASA_1_3/CiscoASA_1_3.yml:1]: @yasta5 @guytamir10 Remove from_xdr_version

File: Packs/JuniperSRX/ModelingRules/JuniperSRX_1_3/JuniperSRX_1_3.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+from_xdr_version: 3.5.0
```
Comment (evisochek) [inline Packs/JuniperSRX/ModelingRules/JuniperSRX_1_3/JuniperSRX_1_3.yml:1]: @yasta5 @guytamir10 Remove from_xdr_version

File: Packs/Mimecast/ModelingRules/MimecastModelingRules_1_3/MimecastModelingRules_1_3.yml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+from_xdr_version: 3.5.0
```
Comment (evisochek) [inline Packs/Mimecast/ModelingRules/MimecastModelingRules_1_3/MimecastModelingRules_1_3.yml:1]: Remove from_xdr_version @yasta5 @guytamir10

File: Packs/ProofpointTAP/ModelingRules/ProofpointTAPModelingRules_1_3/ProofpointTAPModelingRules_1_3.yml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+from_xdr_version: 3.5.0
```
Comment (evisochek) [inline Packs/ProofpointTAP/ModelingRules/ProofpointTAPModelingRules_1_3/ProofpointTAPModelingRules_1_3.yml:1]: @yasta5 @guytamir10 Need to remove from_xdr_version

File: Packs/SafeNet_Trusted_Access/ModelingRules/SafeNetTrustedAccessModelingRules_1_3/SafeNetTrustedAccessModelingRules_1_3.yml [hunk 6]
```diff
@@ -0,0 +1,7 @@
+from_xdr_version: 3.5.0
```
Comment (evisochek) [inline Packs/SafeNet_Trusted_Access/ModelingRules/SafeNetTrustedAccessModelingRules_1_3/SafeNetTrustedAccessModelingRules_1_3.yml:1]: @yasta5 @guytamir10 remove from_xdr_version

File: Packs/Salesforce/ModelingRules/Salesforce_1_3/Salesforce_1_3.xif [hunk 7]
```diff
@@ -0,0 +1,28 @@
+[MODEL: dataset=salesforce_test_raw]
```
Comment (evisochek) [inline Packs/Salesforce/ModelingRules/Salesforce_1_3/Salesforce_1_3.xif:1]: Please provide the real dataset name

File: Packs/Salesforce/ModelingRules/Salesforce_1_3/Salesforce_1_3.yml [hunk 8]
```diff
@@ -0,0 +1,7 @@
+from_xdr_version: 3.5.0
```
Comment (evisochek) [inline Packs/Salesforce/ModelingRules/Salesforce_1_3/Salesforce_1_3.yml:1]: @yasta5 @guytamir10  Remove from_xdr_version

File: Packs/Tableau/ModelingRules/Tableau_1_3/Tableau_1_3.yml [hunk 9]
```diff
@@ -0,0 +1,7 @@
+from_xdr_version: 3.5.0
```
Comment (evisochek) [inline Packs/Tableau/ModelingRules/Tableau_1_3/Tableau_1_3.yml:1]: @yasta5 @guytamir10 remove from_xdr_version

File: Packs/VMwareESXi/ModelingRules/VMwareESXI_1_3/VMwareESXI_1_3.yml [hunk 10]
```diff
@@ -0,0 +1,7 @@
+from_xdr_version: 3.5.0
```
Comment (evisochek) [inline Packs/VMwareESXi/ModelingRules/VMwareESXI_1_3/VMwareESXI_1_3.yml:1]: @yasta5 @guytamir10 remove from_xdr_version

File: Packs/VMwareVCenter/ModelingRules/VMwareVcenter_1_3/VMwareVcenter_1_3.yml [hunk 11]
```diff
@@ -0,0 +1,7 @@
+from_xdr_version: 3.5.0
```
Comment (evisochek) [inline Packs/VMwareVCenter/ModelingRules/VMwareVcenter_1_3/VMwareVcenter_1_3.yml:1]: @yasta5 @guytamir10 Remove from_xdr_version

File: Packs/ZscalerZPA/ModelingRules/ZscalerZPA_1_3/ZscalerZPA_1_3.yml [hunk 12]
```diff
@@ -0,0 +1,7 @@
+from_xdr_version: 3.5.0
```
Comment (evisochek) [inline Packs/ZscalerZPA/ModelingRules/ZscalerZPA_1_3/ZscalerZPA_1_3.yml:1]: @yasta5 @guytamir10 remove from_xdr_version

File: Packs/Salesforce/ModelingRules/Salesforce_1_3/Salesforce_1_3.xif (hunk 7 above)
Comment (guytamir10) [reply Packs/Salesforce/ModelingRules/Salesforce_1_3/Salesforce_1_3.xif:1]: changed from salesforce_test_raw to salesforce_salesforce_raw

File: Packs/BrocadeSwitch/ModelingRules/BrocadeSwitch_1_3/BrocadeSwitch_1_3_schema.json [hunk 13]
```diff
@@ -0,0 +1,8 @@
+{
+  "Brocade_MLX_ICX_raw": {
```
Comment (evisochek) [inline Packs/BrocadeSwitch/ModelingRules/BrocadeSwitch_1_3/BrocadeSwitch_1_3_schema.json:2]: @guytamir10 the name here should the same as the the dataset

File: Packs/SafeNet_Trusted_Access/ModelingRules/SafeNetTrustedAccessModelingRules_1_3/SafeNetTrustedAccessModelingRules_1_3_schema.json [hunk 14]
```diff
@@ -0,0 +1,20 @@
+{
+  "safenet_trusted_access_raw": {
```
Comment (evisochek) [inline Packs/SafeNet_Trusted_Access/ModelingRules/SafeNetTrustedAccessModelingRules_1_3/SafeNetTrustedAccessModelingRules_1_3_schema.json:2]: @yasta5 @guytamir10 The value here should be the same as the dataset

File: Packs/Salesforce/ModelingRules/Salesforce_1_3/Salesforce_1_3_schema.json [hunk 15]
```diff
@@ -0,0 +1,68 @@
+{
+  "salesforce_event_audit_raw": {
```
Comment (evisochek) [inline Packs/Salesforce/ModelingRules/Salesforce_1_3/Salesforce_1_3_schema.json:2]: @yasta5 @guytamir10 The value here should be the same as the dataset

File: Packs/VMwareVCenter/ModelingRules/VMwareVcenter_1_3/VMwareVcenter_1_3_schema.json [hunk 16]
```diff
@@ -0,0 +1,8 @@
+{
+  "VMwareVCenter": {
```
Comment (evisochek) [inline Packs/VMwareVCenter/ModelingRules/VMwareVcenter_1_3/VMwareVcenter_1_3_schema.json:2]: @yasta5 @guytamir10 The value here should be the same as the dataset

**Review by evisochek — APPROVED** [review]

**Pull request merged**